manifests/templates/reconciler-manager-configmap.yaml

```diff
@@ -129,6 +129,18 @@ data:
              requests:
                cpu: "10m"
                memory: "200Mi"
+         - name: gcenode-askpass-sidecar
```

Comment:
I wonder if there're any way we can use this process to delay git-sync errors until the askpass sidecar is up and ready... We don't have a good ordering solution yet for this and it can cause git-sync to error every time the reconciler pod its rescheduled.

Reply:
I can think of a few mitigations but not a real solution to the problem.

For example we could have the reconciler container treat the askpass credential error as a transient error (potentially only during the first minute or so after startup), such that it gets logged but not surfaced on the RSync API.

I think a real solution would require changes to `git-sync` to enable waiting gracefully for the askpass endpoint to be ready.